styles/contact.css

```diff
@@ -0,0 +1,107 @@
+/* Contact section */
+.contact-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(../images/contact-form-m-bg.svg);
+  background-size: contain;
+  background-position: right;
+  background-repeat: no-repeat;
+  padding: 6.4375rem 1.5rem 2.8125rem 1.5rem;
+  width: 100vw;
+  border-top-left-radius: 5.0625rem;
+  background-color: #6070ff;
+}
+
+.contact-wrapper {
+  background-color: #fff;
+}
+
+.contact-wrapper .title {
+  color: white;
+}
+
+.contact-note {
+  font-weight: 400;
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  text-align: center;
+  color: #ebebff;
+  margin-bottom: 5.4375rem;
+}
+
+.contact-wrapper input {
+  height: 3rem;
+  width: 100%;
+  margin-bottom: 1.25rem;
+  padding: 0.9375rem 1rem;
+  font-size: 1.0625rem;
+  color: #172b4d;
+  border: 0.0625rem solid #cfd8dc;
+  border-radius: 0.5rem;
+}
+
+.contact-wrapper textarea {
+  padding: 0.9375rem 1rem;
+  font-size: 0.9375rem;
+  margin-bottom: 1.375rem;
+  width: 100%;
+  height: 10.625rem;
+  border: 0.0625rem solid #cfd8dc;
+  border-radius: 0.5rem;
+  resize: none;
+}
+
+.contact-wrapper input:focus,
+.contact-wrapper textarea:focus {
+  outline: 0.1875rem solid #cfd8dc;
+  outline-offset: 0.125rem;
+}
+
+.contact-wrapper .primary-btn {
+  border: 0.0625rem solid #fff;
+}
+
+.contact-wrapper .primary-btn:active {
+  background-color: #2230d2;
+  border: none;
+}
+
+/* Media Section for the Footer */
+
+@media (min-width: 768px) {
+  .contact-card {
+    width: 100vw;
+    background-image: url(../images/contact-form-d-bg.svg);
+    background-origin: padding-box;
+    background-position: center;
+    gap: 30px;
+  }
+
+  .contact-wrapper .title {
+    width: 100%;
+    text-align: center;
+  }
+
+  .contact-note {
+    margin-bottom: 0;
+    width: 569px;
+  }
+
+  .contact-wrapper form {
+    display: flex;
+    flex-direction: column;
+    gap: 27px;
+    width: 45%;
+  }
```

Comment:
- [ ] [Optional] I would suggest you decrease the width and make it `30%`  So it match the design

Reply:
Done, thank you for your feed back 😄  @roshan-bajgain